apps/examples/board_specific_demo/ameba_secure_world_demo/secure_world.c

```diff
@@ -0,0 +1,77 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+
+#define SHOW_USAGE printf("Usage: secure_world -a secure_world_address -r/-w\n \
+		-r: run the read test\n \
+		-w: run the write test\n \
+		-a secure_world_address: take the address as input\n \
+		Valid Range: 0x7034C020 - 0x7039C000\n \
+		The default address when no input address is given is 0x735C021");
+
+#ifdef  CONFIG_AMEBALITE_TRUSTZONE
+volatile uint32_t  *address = 0x735C021;
+#endif
+
+/****************************************************************************
+ * secure_world_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int secure_world_main(int argc, char *argv[])
+#endif
+{
+	int option;
+	volatile int value;
+	bool r_flag = false, w_flag = false;
+	while ((option = getopt(argc, argv, "a:rw")) != -1) {
+		switch (option) {
+		case 'a':
+			sscanf(optarg, "%x", &address);
+			break;
+		case 'r':
+			r_flag = true;
+			break;
+		case 'w':
+			w_flag = true;
+			break;
+		default:
+			printf("INVALID INPUT\n");
+			SHOW_USAGE
+			return 0;
+		}
+	}
+	if (!r_flag && !w_flag)
```

Comment:
1. Need brackets even it's a single statement
2. closing and else should be at the same line
```
if (...) {
   do
} else if {
} else {
}
```